packages/react/src/components/primitive/root.tsx

```
@@ -0,0 +1,317 @@
+}) {
+  if (!children) return defaultChildren;
+  return typeof children === "function"
+    ? children?.(usePrimitiveValues(componentName))
+    : children;
+}
+
+export type HasDisplayName = {
+  displayName: string;
+};
+
+export type RefProp<T extends Function> = T extends (
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)